tests/testthat/test-panels.R

```diff
@@ -3,6 +3,7 @@ context("Panel background")
 p1 <- ggplot() +
   geom_point(aes(Sepal.Length, Sepal.Width,
                  colour = Species, size = Species), data = iris) +
+  theme_grey() + 
```

Comment:
ok well if the shiny test works on your machines then that is fine with me
+1

On Wed, Jul 15, 2015 at 11:57 AM, Kevin Ferris <notifications@github.com>
wrote:

> In tests/testthat/test-panels.R
> <https://github.com/tdhock/animint/pull/74#discussion_r34694464>:
>
> > @@ -3,6 +3,7 @@ context("Panel background")
> >  p1 <- ggplot() +
> >    geom_point(aes(Sepal.Length, Sepal.Width,
> >                   colour = Species, size = Species), data = iris) +
> > +  theme_grey() +
>
> @tdhock <https://github.com/tdhock> This should take care of the
> theme_bw() issue
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tdhock/animint/pull/74/files#r34694464>.
>